packages/terra-alert/src/Alert.module.scss

```diff
@@ -40,11 +44,16 @@
     padding-top: 0;
   }
 
-  .body-narrow {
+  .alert-parent-tiny.narrow {
     margin-bottom: 0;
   }
 
-  .body-std {
+  .alert-parent-tiny,
+  .alert-parent-small,
+  .alert-parent-medium,
+  .alert-parent-large,
+  .alert-parent-huge,
+  .alert-parent-enormous {
     margin-bottom: var(--terra-alert-std-margin-bottom, 0.7143rem);
     margin-right: var(--terra-alert-std-margin-right, 0.6667rem);
```

Comment:
As is, we shouldn't be applying margin-right to the narrow alert that has an actin or dismiss:
https://github.com/cerner/terra-core/pull/2747/files#diff-e3a10ba555be3590e95544911b047ebcL148

Reply:
By changing the classes that use these variables, does that mean we should be introducing a new variables to reflect the new classes in the name?